test.sh

```diff
@@ -55,7 +53,7 @@ while [[ $# -gt 0 ]]; do
 			echo "test.sh - Run the tests for dd-trace-go"
 			echo "	this script requires gotestsum, goimports, docker and docker-compose."
 			echo "	-l | --lint		- Run the linter"
-			echo "	-a | --appsec		- Test with appsec enabled"
+			echo "	--appsec		- Test with appsec enabled"
```

Comment:
Revert needed to align with some other parts of this script:
```suggestion
			echo "	-a | --appsec		- Test with appsec enabled"
```